CONTRIBUTING.RELEASE.md

```diff
@@ -0,0 +1,247 @@
+# OpenTofu release manual
+
+> [!WARNING]
+> This manual is intended for OpenTofu core and fork maintainers. If you are looking for the normal contribution guide, see [this file](CONTRIBUTING.md).
+
+This manual describes how to create an OpenTofu release. OpenTofu has two kinds of releases. Alpha releases are created
+from the `main` branch, while we split off a version (e.g. `v1.8`) branch before creating a `beta`, `rc` or `stable`
+release.
+
+---
+
+## Gathering the team for a release
+
+To create a release, make sure you have people on standby with the following credentials:
+
+- Cloudflare
+- PackageCloud
+- Snapcraft
+- Linkedin
+- X
+
+---
+
+## Preparing public relations collaterals for a release
+
+Before you start creating a release, make sure you have the following marketing collaterals ready to be published:
+
+<details><summary>
+
+### Alpha version (`X.Y.0-alphaW`)
+
+</summary>
+
+- Feature Preview video for upcoming flagship features (see https://www.youtube.com/@opentofu for examples)
+- "Help us test..." blog post (see https://opentofu.org/blog/ for examples)
+- Community Slack announcement
+- Linkedin and X posts
+
+</details>
+
+<details><summary>
+
+### Beta (`X.Y.0-betaW`)
+
+</summary>
+
+- "Get ready for..." blog post (see https://opentofu.org/blog/ for examples)
+- Community Slack announcement
+- Linkedin and X posts
+
+</details>
+
+<details><summary>
+
+### Release Candidate (`X.Y.0-rcW`)
+
+</summary>
+
+- Community Slack announcement
+- Linkedin and X posts
+
+</details>
+
+<details><summary>
+
+### Stable release (`X.Y.0`)
+
+</summary>
+
+- Release blog post with the feature and community highlights since the last release (see https://opentofu.org/blog/ for examples)
+- Community Slack announcement
+- Linkedin and X posts
+
+</details>
+
+<details><summary>
+
+### Point release (`X.Y.Z`)
+
+</summary>
+
+- Community Slack announcement
+
+</details>
+
+---
+
+## Preparing the repository for a release
+
+Before you can create a release, you need to make sure the following files are up to date:
+
+- [CHANGELOG.md](CHANGELOG.md) (Note: do not remove the `(unreleased)` string from the version number before the stable release.)
+- [version/VERSION](version/VERSION)
+
+Ideally, make sure these changes go in as the last PR before the release.
+
+---
+
+## Tagging the release
+
+Now that you have the files up to date, do the following:
+
+1. On your computer, make sure you have checked the correct branch:
+   * `main` for `alpha` releases
+   * `vX.Y` for any other releases (assuming you are releasing version `X.Y.Z`)
+2. Make sure the branch is up-to-date by running `git pull`
+3. Create the correct tag: `git tag -m "X.Y.Z" vX.Y.Z` (assuming you are releasing version `X.Y.Z`) 
+   * If you have a GPG key, consider adding the `-s` option to create a GPG-signed tag
+4. Make sure there are no locally-created unnecessary tags by running `git tag`
+5. Push the tag: `git push --tags`
```

Comment:
I think this could be reduced to a single step that avoids the risk of pushing anything other than this one tag:

> 4. Push the tag: `git push vX.Y.Z`